### PR TITLE
[TMVA] CV - Evaluate single- + multiprocessing

### PR DESCRIPTION
**Pull request by ashlaban**
Uses features introduced in root-project/root#1869, can be merged when that one is integrated.

### CODE REVIEW

File: root/tmva/tmva/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+RB_ADD_GBENCHMARK(CrossValidationBenchmarks
+  CrossValidationBenchmarks.cxx
+  LABEL short
+  LIBRARIES Core tree MathCore TMVA)
```
[the remark this answers is not in the document]
Comment (ashlaban) [reply root/tmva/tmva/CMakeLists.txt]: 👍

**Comment by oshadura** [comment]
@ashlaban sorry for red light in Travis, it is my fault (we have a missing dependency for TMVA)

**Comment by ashlaban** [comment]
This should now be ready for integration! Can I trigger a rebuild?

**Comment by oshadura** [comment]
@ashlaban builds are re-triggered, after green light they will be in master ;)

**Comment by oshadura** [comment]
@ashlaban finally benchmark is compiling properly so I merged it in https://github.com/root-project/rootbench/pull/70. We can close this PR..

**Comment by ashlaban** [comment]
Thanks for your hard work @oshadura!

**Pull request closed without merging**